extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -127,263 +138,466 @@ For example
-              }
-            }
-          }
-        }
-      },
-      "propertyTables": [{
-        "name": "tree",
-        "class": "tree",
-        "count": 10,
-        "properties": {
-          "height": {
-            "bufferView": 0
-          },
-          "birdCount": {
-            "bufferView": 1
-          },
-          "species": {
-            "bufferView": 2,
-            "stringOffsetBufferView": 3
-          }
-        }
-      }]
-    }
-  }
-}
-```
-
-`class` is the ID of the class in the schema. `count` is the number of features in the property table, as well as the length of each property array. Property arrays are stored in glTF buffer views and use the binary encoding defined in the [Table Format](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata#table-format) section of the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata).
+Each property table defines a specified number (`count`) of features conforming to a particular class (`class`), with property values stored as parallel arrays in a column-based binary layout.
+
+Property tables are defined as entries in the `propertyTables` array of the root-level `EXT_mesh_features` extension, and may be referenced by extensions on primitive or node objects.
+
+The property table may provide value arrays for only a subset of the properties of its class, but class properties marked `required: true` must not be omitted. Each property value array given by the property table must be defined by a class property with the same alphanumeric property ID, with values matching the data type of the class property.
+
+> **Example:** A `tree_survey_2021-09-29` property table, implementing the `tree` class defined in earlier examples. The table contains observations for 10 trees, with each property defined by a buffer view containing 10 values.
+>
+> ```jsonc
+> {
+>   "extensions": {
+>     "EXT_mesh_features": {
+>       "schema": { ... },
+>       "propertyTables": [{
+>         "name": "tree_survey_2021-09-29",
+>         "class": "tree",
+>         "count": 10,
+>         "properties": {
+>           "height": {
+>             "bufferView": 0
+>           },
+>           "birdCount": {
+>             "bufferView": 1
+>           },
+>           "species": {
+>             "bufferView": 2,
+>             "stringOffsetBufferView": 3
+>           }
+>         }
+>       }]
+>     }
+>   }
+> }
+> ```
+
+Property arrays are stored in glTF buffer views and use the binary encoding defined in the [Table Format](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata#table-format) section of the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata).
 
 As in the core glTF specification, values of NaN, +Infinity, and -Infinity are never allowed.
 
 Each buffer view `byteOffset` must be aligned to a multiple of 8 bytes.
```

Comment:
There should probably be a short paragraph about why this extension enforces 8 byte alignment - i.e. to support 64-bit types without requiring a copy on the client. The glb rules are pretty clear but there's not enough justification.

Reply:
Hm, I feel like in practice we've been acting like it's (b) as this is more consistent (no need to check the data type first, just check if `offset % 8 === 0`. though technically speaking only (c) is strictly necessary when I think about it...